### ISSUE TITLE
Capacity reference is missing

### ISSUE THREAD

**Issue by hiroki-o** [Unity-Technologies/AssetGraph#121]
**[Original report](https://bitbucket.org/Unity-Technologies/assetbundlegraphtool/issue/86) by Aleksandr Ilarionov (Bitbucket: [aleksandr_ilarionov](https://bitbucket.org/aleksandr_ilarionov), ).**

----------------------------------------

1) dev-2 commit has fatal error.

Assets/AssetBundleGraphTool/UnityEngine.AssetGraph/Editor/GUI/GraphView/AssetReferencePort.cs(10,82): error CS0246: The type or namespace name `Capacity' could not be found. Are you missing an assembly reference?

2) Removing Capacity reference from everywhere it's used fixes the issue.

Just thought if you forgot to commit the Capacity.cs file with the rest of the code??

Hope that helps, big fan of this tool, good job!!!

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

@aleksandr_ilarionov Thank you for reporting. dev-2.0 branch is a temporary branch for version 2.0 works, and it is very early right now. Please expect nothing will work correctly for next a few months :)
This branch will be merged into dev once refactoring is done, and will be removed when merge happens.

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

And thank you for kind words and glad that you like the tool! It really helps me keep up working on this.
